diff --git a/cmake/conan_build.cmake b/cmake/conan_build.cmake
index 3bf3a72..c1d875a 100644
--- a/cmake/conan_build.cmake
+++ b/cmake/conan_build.cmake
@@ -2,15 +2,15 @@ list(APPEND CMAKE_PREFIX_PATH "${CMAKE_BINARY_DIR}")
 
 if(BTCPP_GROOT_INTERFACE)
     find_package(ZeroMQ REQUIRED)
-    list(APPEND BTCPP_EXTRA_LIBRARIES ${ZeroMQ_LIBRARIES})
-    list(APPEND BTCPP_EXTRA_INCLUDE_DIRS ${ZeroMQ_INCLUDE_DIRS})
-    message(STATUS "ZeroMQ_LIBRARIES: ${ZeroMQ_LIBRARIES}")
+    # list(APPEND BTCPP_EXTRA_LIBRARIES ${ZeroMQ_LIBRARIES})
+    # list(APPEND BTCPP_EXTRA_INCLUDE_DIRS ${ZeroMQ_INCLUDE_DIRS})
+    # message(STATUS "ZeroMQ_LIBRARIES: ${ZeroMQ_LIBRARIES}")
 endif()
 
 if(BTCPP_SQLITE_LOGGING)
     find_package(SQLite3 REQUIRED)
-    list(APPEND BTCPP_EXTRA_LIBRARIES ${SQLite3_LIBRARIES})
-    message(STATUS "SQLite3_LIBRARIES: ${SQLite3_LIBRARIES}")
+    # list(APPEND BTCPP_EXTRA_LIBRARIES ${SQLite3_LIBRARIES})
+    # message(STATUS "SQLite3_LIBRARIES: ${SQLite3_LIBRARIES}")
 endif()
 
 
